src/app/lib/styledGrid/index.jsx

```diff
@@ -21,11 +22,20 @@ export const Grid = styled.div`
   ${layoutGridWrapper};
 `;
 
+export const GridWithPadding = styled.div`
+  ${layoutGridWrapper};
+  padding-bottom: ${GEL_SPACING_QUAD};
+`;
+
 export const GhostGrid = styled.div`
```

Comment:
Does anything still use these?

Reply:
Nothing in production, only storybook stories. We will likely need `Grid` for nested grids shortly.

Reply:
Implementation changed to add the padding directly onto the grid.